recipes/sphinx-issues/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: sphinx-issues
+  version: {{ version }}
+
+source:
+  fn: sphinx-issues-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/sphinx-issues/sphinx-issues-{{ version }}.tar.gz
+  md5: 6065e4538b8ead02e35a777711eab6f5
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sphinx
```

Comment:
Not normally during the build. The flag `--single-version-externally-managed` is suppose to stop install time dependencies from being pulled in. Setup time dependencies is a different story. Though that doesn't appear to be the [case]( https://github.com/sloria/sphinx-issues/blob/e57049479faddc79af0c28fc8cc431669e0374ba/setup.py#L36 ) here.

The main issue is we have found parts of `.pyc` files and such that don't belong that have caused some serious issues ( https://github.com/conda-forge/conda-smithy/issues/289 ).